copier/types.py

```diff
@@ -21,5 +22,6 @@
 
 # miscellaneous
 CheckPathFunc = Callable[[StrOrPath], bool]
+Choices = Union[Sequence[AnyOrTupleAny], Dict[Any, Any]]
```

Comment:
You can't specify a list's length by static typing because they're mutable: https://stackoverflow.com/a/44833864/1468388

So the closest you can get is:

```suggestion
Choices = Union[List[Any], Tuple[Any, Any], Dict[Any, Any]]
```

Reply:
I wanted to represent the possibility for the list to be composed of both strings and tuples of 2 strings.

Reply:
Yes, you're right.

Also, after all, YAML will never send us a tuple. It always evaluates to lists. So chances are to get a list or a dict.

Reply:
Done.